scripts/setup/vue.sh

```diff
@@ -14,17 +14,87 @@ else
+
+
+# set scripts
+sed 's/"scripts": {/"scripts": {\
+  \  "buildConfig":"webpack --entry .\/src\/luigi-config\/luigi-config.es6.js -o .\/public\/assets\/luigi-config.js --mode production",/1' package.json > p.tmp.json && mv p.tmp.json package.json
+
+
+echo "const webpack = require('webpack');
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+module.exports = {
+  pages: {
+    sampleapp: {
+      entry: 'src/main.js',
+      template: 'public/sampleapp.html',
+      filename: 'sampleapp.html'
+    }
+  },
+  lintOnSave:true,
+  runtimeCompiler: true,
+  outputDir: 'dist',
+  configureWebpack: {
+    module: {
+      rules: [
+        {
+          test: /\.css$/,
+          use: ['css-loader']
+        },
+        {
+          test: /\.scss$/,
+          use: ['sass-loader']
+        }
+      ]
+    },
+    plugins: [
+      new CopyWebpackPlugin(
+        [
+          {context:'public',to:'index.html',from:'index.html'},
+          {context:'node_modules/@luigi-project/core',to:'./luigi-core',from:{glob:'**',dot:true}},
+          {context:'node_modules/@luigi-project/client',to:'./luigi-client',from:{glob:'**',dot:true}},
+          {
+            from: 'node_modules/fundamental-styles/dist',
+            to: './fundamental-styles'
+          },
+          {
+            from: 'node_modules/@sap-theming/theming-base-content',
+            to: './fonts'
+          }
+        ],
+        {ignore:['.gitkeep','**/.DS_Store','**/Thumbs.db'],debug:'warning'}
+      )]
+    }
+  };" > vue.config.js
+
+npm run buildConfig
 
-vue create -d my-original-vue-app && cd my-original-vue-app
-npm i -save @luigi-project/core @luigi-project/client fiori-fundamentals
-mkdir -p public/assets
-mv public/index.html public/vue.html
-curl https://raw.githubusercontent.com/SAP/luigi/master/core/examples/luigi-example-vue/public/index.html > public/index.html
-sed 's/luigi-config.js/assets\/luigi-config.js/g' public/index.html > public/index.tmp.html && mv public/index.tmp.html public/index.html
-curl https://raw.githubusercontent.com/SAP/luigi/master/test/e2e-test-application/src/luigi-config/basic/basicConfiguration.js > public/assets/luigi-config.js
-curl https://raw.githubusercontent.com/SAP/luigi/master/test/e2e-test-application/src/assets/sampleexternal.html > public/assets/basicexternal.html
-echo "const webpack=require('webpack');const CopyWebpackPlugin=require('copy-webpack-plugin');module.exports={pages:{sampleapp:{entry:'src/main.js',template:'public/vue.html',filename:'vue.html'}},lintOnSave:true,runtimeCompiler:true,outputDir:'dist',configureWebpack:{plugins:[new CopyWebpackPlugin([{context:'public',to:'index.html',from:'index.html'},{context:'node_modules/@luigi-project/core',to:'./luigi-core',from:{glob:'**',dot:true}},{context:'node_modules/@luigi-project/client',to:'./luigi-client',from:{glob:'**',dot:true}}],{ignore:['.gitkeep','**/.DS_Store','**/Thumbs.db'],debug:'warning'})]}};" > vue.config.js
```

Comment:
![Bildschirmfoto 2020-05-18 um 14 32 17](https://user-images.githubusercontent.com/14272184/82213466-76d09c00-9914-11ea-9ce7-9e8a2852a03d.png)
Vue does not work anymore for me

Reply:
It still works for me, did you serve the `..scripts/setup/assets/index.html` from localhost? .. or at least copy it after the curls, right in the beginning of vue startup manually

Reply:
yes, I did. 
Now it works.. :). All good :)